opentreemap/treemap/templates/treemap/map_feature_detail.html

```diff
@@ -32,13 +32,25 @@
         <div class="modal-dialog">
             <div class="modal-content lightbox">
                 <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+                <p class="alert alert-danger" data-class="delete">
+                    {% trans "Are you sure you want to delete this photo?" %}
+                </p>
                 <img data-photo-image src="">
                 <div class="lightbox-caption">
+                    <button class="btn btn-small btn-danger pull-right" type="button" disabled="disabled"
+                            data-class="view"
+                            data-photo-delete-start
+                            data-disabled-title="{% trans 'Deleting of photos is not available to all users' %}"
+                            >{% trans "Delete" %}</button>
                     <a class="btn btn-small pull-right" data-class="view" data-photo-edit>{% trans "Edit" %}</a>
```

Comment:
Placing this button inside a `<div class="pull-right">` along with the "Edit" button, and removing the `pull-right` class from the buttons themselves, and doing the same for the "Yes" and "No" buttons below, will give them some padding so they don't butt up against each other:
```diff
diff --git a/opentreemap/treemap/templates/treemap/map_feature_detail.html b/opentreemap/treemap/templates/treemap/map_feature_detail.html
index e96dae8..9bdaa21 100644
--- a/opentreemap/treemap/templates/treemap/map_feature_detail.html
+++ b/opentreemap/treemap/templates/treemap/map_feature_detail.html
@@ -37,20 +37,24 @@
                 </p>
                 <img data-photo-image src="">
                 <div class="lightbox-caption">
-                    <button class="btn btn-small btn-danger pull-right" type="button" disabled="disabled"
-                            data-class="view"
-                            data-photo-delete-start
-                            data-disabled-title="{% trans 'Deleting of photos is not available to all users' %}"
-                            >{% trans "Delete" %}</button>
-                    <a class="btn btn-small pull-right" data-class="view" data-photo-edit>{% trans "Edit" %}</a>
+                    <div class="pull-right">
+                        <a class="btn btn-small" data-class="view" data-photo-edit>{% trans "Edit" %}</a>
+                        <button class="btn btn-small btn-danger" type="button" disabled="disabled"
+                                data-class="view"
+                                data-photo-delete-start
+                                data-disabled-title="{% trans 'Deleting of photos is not available to all users' %}"
+                                >{% trans "Delete" %}</button>
+                    </div>
                     <a class="btn btn-small pull-left btn-rotate" data-class="edit" data-photo-rotate="-90"><i class="icon-ccw"></i></a>
                     <a class="btn btn-small pull-left btn-rotate" data-class="edit" data-photo-rotate="90"><i class="icon-cw"></i></a>
                     <button data-class="edit" disabled="disabled" class="btn btn-small pull-right"
                             data-photo-save="">{% trans "Save" %}</button>
-                    <button data-class="delete" disabled="disabled" class="btn btn-small btn-danger pull-right"
-                            data-photo-confirm="">{% trans "Yes" %}</button>
-                    <button data-class="delete" class="btn btn-small pull-right"
-                            data-photo-keep="">{% trans "No" %}</button>
+                    <div class="pull-right">
+                        <button data-class="delete" class="btn btn-small"
+                                data-photo-keep="">{% trans "No" %}</button>
+                        <button data-class="delete" disabled="disabled" class="btn btn-small btn-danger"
+                                data-photo-confirm="">{% trans "Yes" %}</button>
+                    </div>
                 </div>
             </div>
         </div>
```